src/components/__tests__/__fixtures__/sampleFlaws.ts

```diff
@@ -0,0 +1,82 @@
+import { type ZodFlawType } from '@/types/zodFlaw';
+import { sampleFlawComments_1 } from './sampleFlawComments';
+import { sampleFlawAffects_1 } from './sampleFlawAffects';
+import { sampleFlawCvssScores_1 } from './sampleFlawCvssScores';
+
```

Comment:
I think having a function is helpful in cases were a sample flaw could be mutated in a test.